scripts/setup-release-on-ubuntu.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+RELEASE_VERSION=${RELEASE_VERSION:=0.24.0}
+RELEASE_FILE="https://github.com/aeternity/epoch/releases/download/v${RELEASE_VERSION}/epoch-${RELEASE_VERSION}-ubuntu-x86_64.tar.gz"
+TEMP_RELEASE_FILE=${TEMP_RELEASE_FILE:=/tmp/epoch.tgz}
+TARGET_DIR=${TARGET_DIR:=$HOME/epoch}
+EPOCH_CONFIG=${EPOCH_CONFIG:=$TARGET_DIR/epoch.yaml}
+NPROC=$(nproc)
+
+echo ""
+read -sp "Please provide a secret key passphrase to be used for your keys: " keypassphrase
+echo ""
+
+echo -e "\nPrepare host system and install dependencies...\n"
+sudo apt-get update
+sudo apt-get upgrade -y
+sudo apt-get install -y curl libsodium23
```

Comment:
Not sure with what openssl version the WSL ubuntu ships, but the `epoch` require `sudo apt-get install libssl1.0.0` on ubuntu 18.04.

Could you please consider adding this requirement as well (`sudo apt-get install libssl1.0.0`) so that the script also runs in official ubuntu18 distros?

I tested it on `docker run -it ubuntu:18.04`, it doesn't run without libssl1.0.0

Reply:
I did test on WSL and all of the things did work as expected, epoch did start, was mining and syncing with testnet. My WSL installation was completely fresh.

Reply:
I'll add it to the set of packages.